app/Models/SelfServiceAccessOverrideEligibility.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace App\Models;
```

Comment:
I'm not sure how I feel about this living in Models when it's not in the database. I don't have a better suggestion though.

Reply:
Same on both counts. I think it's fine.